src/NServiceBus.Core/UnitOfWork/TransactionScopes/TransactionScopeUnitOfWorkBehavior.cs

```diff
@@ -28,5 +28,16 @@ public async Task Invoke(IIncomingPhysicalMessageContext context, Func<IIncoming
         }
 
         readonly TransactionOptions transactionOptions;
+
+        public class Registration : RegisterStep
+        {
+            public Registration(TransactionOptions transactionOptions) : base("HandlerTransactionScopeWrapper",
+                typeof(TransactionScopeUnitOfWorkBehavior),
+                "Makes sure that the handlers gets wrapped in a transaction scope",
+                b => new TransactionScopeUnitOfWorkBehavior(transactionOptions))
+            {
+                InsertAfter("ExecuteUnitOfWork");
```

Comment:
By making sure it is inserted after `ExecuteUnitOfWork` the 7.2 behavior is restored

Reply:
I'm getting this very behavior in my code.  We are on 7.3.

"Ambient transaction detected. The transaction scope unit of work is not supported when there already is a scope present."

You indicate the change in between 7.2 to 7.3 cause TransactionScopeUnitOfWorkBehavior to be placed too early in the pipeline.  We definitely have custom unit of works in play here.  How might I workaround this behavior in 7.3, if possible?